# Задание №1

# Функция plus_two() выполняет одну простую задачу — выводит результат сложения переданного
#  в нее числа 2 и значения переменной number. В переменную number должно быть передано число.
#  Обработайте ситуацию, если в эту переменную переданы данные какого-то другого типа. 
# В случае ошибки напечатайте в консоли сообщение «Ожидаемый тип данных — число!».
# Запустите код и проверьте работу кода в консоли.
# Подсказка:
# Используйте конструкцию try/except.В процессе поиска решения попробуйте вывести в консоль сумму строки и числа, изучите сообщение об ошибке. В Python есть специальное исключение для ситуации, если тип переданного значения не соответствует ожиданиям.    
def plus_two(a,b):
    number=a+b
    return number

try:
    a,b=map(int,input("Введите два число:").split())
    number =plus_two(a,b)
    print(number)
except:
    print("Ожидаемый тип данных — число")
# Задание №2
# Напишите программу, которая позволяет получить доступ к элементу массива,
#  индекс которого выходит за границы, и обработаем соответствующее исключение.
array=[1,2,3,4,5]
try:
    i=int(input("Введите индекс элемента массива: "))
    print(array[i])
except IndexError:
    print("Вы вышли заграницу массива")
except:
    print("Ожидаемый тип данных — число")